packages/terra-card/src/CardBody.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+  /**
+   * Provides 10px of themeable padding vertical
+   */
+  hasPaddingVertical: PropTypes.bool,
+  /**
+   * Provides 10px of themeable padding horizontal
```

Comment:
Nit: It might make sense to remove value / unit specs from documentation since it's value can be themed to anything making the documentation potentially incorrect. ( If you make this change it might be worth removing the double spacing on lines 6/7 and 29/30)